### ISSUE TITLE
Issue with glob in Notebook for 'deaddisk --add_windows_directory' on v0.6.4-rc3

### ISSUE THREAD

**Issue by teddy-ROxPin** [Velocidex/velociraptor#1717]
When I attempt to use a glob further than just 'C:/' I am met with an error when using a Notebook after a gui launch using a remap file output from 'deaddisk --add_windows_directory'

This works without error:
```
SELECT * FROM glob(globs='C:/')
```

The following thows an error of `Globber: open /ouput/C\: The system cannot find the path specified. while processing C:`
```
SELECT * FROM glob(globs='C:/Users')
```

Drive remapping section from the 'deaddisk --add_windows_directory' output I'm using (yes, I forgot the 't' in output :) ):
```
- type: mount
  description: 'Mount the directory .\ouput\C\ on the C: drive (NTFS)'
  from:
    accessor: file
    prefix: .\ouput\C\
  "on":
    accessor: ntfs
    prefix: '\\.\C:'
    path_type: ntfs
- type: mount
  description: 'Mount the directory .\ouput\C\ on the C: drive (FILE Accessor)'
  from:
    accessor: file
    prefix: .\ouput\C\
  "on":
    accessor: file
    prefix: 'C:'
    path_type: windows
- type: mount
  description: 'Mount the directory .\ouput\C\ on the C: drive (Auto Accessor)'
  from:
    accessor: file
    prefix: .\ouput\C\
  "on":
    accessor: auto
    prefix: 'C:'
    path_type: windows
```

**Comment by scudette** [comment]
The prefix can not be a relative path (relative to what?) it should be the full path relative to the root of the mapping - maybe this is what is getting confused here.

The deaddisk command should convert the path to an absolute path.

**Comment by teddy-ROxPin** [comment]
When I changed the prefix to an absolute path this error went away.

That section from above was an unedited output from the 'deaddisk --add_windows_directory' command. It created those prefix paths relative to where 'velociraptor-v0.6.4-rc3-windows-amd64.exe' was, that's also where I ran the deaddisk command from.

**Comment by scudette** [comment]
Should be fixed by 0.6.4 - the path will now be converted to an absolute path automatically.
